libcob/java.c

```diff
@@ -79,11 +79,15 @@ resolve_java (const char		*class_name,
 	cls = (*env)->FindClass(env, jni_class_name);
 	free(jni_class_name);
 	if (!cls) {
+		(*env)->ExceptionDescribe(env);
+		(*env)->ExceptionClear(env);
```

Comment:
fine during development, needs to be adjusted later - see review on the other PR